### PR TITLE
Add select options to case summary view

### PR DESCRIPTION
**Pull request by proteusvacuum**
For multiple choice questions, you can now see the possible values that will save to the case. Hooray! 

screenshot from 2017-02-03 10-55-14

@czue @kaapstorm 
#INChackathon

### CODE REVIEW

**Comment by orangejenny** [comment]
nice

**Comment by czue** [comment]
wow this is amazing and so little code!

File: corehq/apps/app_manager/templates/app_manager/ng_partials/case_summary_view.html [hunk 1]
```diff
@@ -187,6 +187,14 @@ <h1 style="font-size: 1.8em;">{% trans "Case Summary" %}</h1>
     <li ng-repeat="question in questions">
         {% verbatim %}<i class="hq-icon fa-li {{ getIcon(question.question.type) }}"></i>{% endverbatim %}
         {% verbatim %}<span ng-show="showLabels">{{ getQuestionLabel(question.question) }}</span>{% endverbatim %}
+        {% verbatim %}
+        <ul ng-if="question.question.options">
+            <li ng-repeat="option in question.question.options">
+                {{ getSelectOptionsLabel(option) }}
```
Comment (czue) [inline corehq/apps/app_manager/templates/app_manager/ng_partials/case_summary_view.html]: does this respect the question IDs toggle? was looking at the screenshot and just thinking IDs are often more useful than labels.

**Comment by snopoke** [comment]
Nice, loving the improvements that are getting added to app summary!

File: corehq/apps/app_manager/templates/app_manager/ng_partials/case_summary_view.html (hunk 1 above)
Comment (proteusvacuum) [inline corehq/apps/app_manager/templates/app_manager/ng_partials/case_summary_view.html]: good point: https://github.com/dimagi/commcare-hq/pull/14760

**Pull request merged**